inlang/source-code/sdk/src/createNodeishFsWithAbsolutePaths.test.ts

```diff
@@ -21,6 +21,14 @@ it("intercepts paths correctly for readFile", async () => {
 		["../folder/file.txt", `/Users/samuel/Documents/paraglide/folder/file.txt`],
 		["../../file.txt", `/Users/samuel/Documents/file.txt`],
 		["../../../file.txt", `/Users/samuel/file.txt`],
+		["..\\file.txt", `/Users/samuel/Documents/paraglide/file.txt`],
+		["..\\folder\\file.txt", `/Users/samuel/Documents/paraglide/folder/file.txt`],
+		["..\\..\\file.txt", `/Users/samuel/Documents/file.txt`],
+		["..\\..\\..\\file.txt", `/Users/samuel/file.txt`],
```

Comment:
I don't think windows "go up to parent directory" paths use `\\..\..`. An initial search supports my impression https://stackoverflow.com/questions/5050525/what-are-and-in-windows-directory#:~:text=Use%20two%20consecutive%20periods%20(..,For%20more%20information%2C%20see%20Paths.

Reply:
It's just a test which should properly resolve & I don't cover Windows standard behavior here. It can be human error & therefore I want to guarantee it's resolving correctly even with backslashs.

Reply:
fixed with normalizing path before.